src/auxiliary/Debug.cc

```diff
@@ -250,13 +250,14 @@ void Debug::printTilesMaps(BaseMatrix<scalar_t> const& A)
 ///  - "i" if invalid
 /// Second char:
 ///  - "h" if on hold
-///  - " " otherwise
+///  - "_" otherwise
 ///
 /// Prints aditional char for Layout:
```

Comment:
Additional.

Reply:
Also, I just notice it is missing the char `u` and `x` in the description of the function.

Reply:
Seems there was a discrepancy between CPU and GPU: "w" for workspace on CPU, "x" on GPU. I changed "x" to "w" for consistency.
"u" is documented below, though again there's a discrepancy that "u" isn't printed on the GPU. I'm not familiar enough with this to know why that is.

Later, I think we could merge the CPU and GPU codes here to minimize code and avoid discrepancies.